sample/src/main/res/values/strings.xml

```diff
@@ -63,5 +64,7 @@
 
     <!-- WMTS Capabilities Url -->
     <string name="wmts_capabilities_url">https://tpwd.texas.gov/arcgis/rest/services/Vegetation_Mapping/Texas_Ecological_Mapping_Systems_Data/mapserver/WMTS/1.0.0/WMTSCapabilities.xml</string>
+    <string name="title_activity_configurable_location_circle">ConfigurableLocationCircleActivity
```

Comment:
Currently this is not functional. It is not used in the actual title rendering and thus not required

Reply:
Done.